core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupTemplate.java

```diff
@@ -16,12 +21,7 @@
 import org.jboss.logmanager.errormanager.OnlyOnceErrorManager;
 import org.jboss.logmanager.formatters.ColorPatternFormatter;
 import org.jboss.logmanager.formatters.PatternFormatter;
-import org.jboss.logmanager.handlers.AsyncHandler;
-import org.jboss.logmanager.handlers.ConsoleHandler;
-import org.jboss.logmanager.handlers.FileHandler;
-import org.jboss.logmanager.handlers.PeriodicRotatingFileHandler;
-import org.jboss.logmanager.handlers.PeriodicSizeRotatingFileHandler;
-import org.jboss.logmanager.handlers.SizeRotatingFileHandler;
+import org.jboss.logmanager.handlers.*;
```

Comment:
Nice work @Sparow199 . This is a general comment on wildcard imports which we tend not to use in the codebase.

Reply:
I know it, i'm using Intellij so... it's the default behavior.